src/components/Modal.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export default function Modal({ message, showModal, setShowModal }) {
```

Comment:
for consistency reasons for components it would be better to move the export default to the end of the function